src/cli.ts

```diff
@@ -15,87 +16,166 @@ const defaultCommand = define({
       type: 'string',
       default: 'auto',
       description: `Package manager to use for packing ('auto' | 'npm' | 'yarn' | 'pnpm' | 'bun')`
+    },
+    'list-tarball-files': {
+      type: 'boolean',
+      default: false,
+      description: 'List all files in the tarball',
     }
   },
   async run(ctx) {
     const root = ctx.positionals[0];
     let pack = ctx.values.pack as PackType;
+    const showAllFiles = ctx.values['list-tarball-files'] as boolean;
 
     prompts.intro('Generating report...');
 
     if (typeof pack === 'string' && !allowedPackTypes.includes(pack)) {
       prompts.cancel(
         `Invalid '--pack' option. Allowed values are: ${allowedPackTypes.join(', ')}`
       );
-      process.exit(1);
+      throw new Error('Invalid --pack option');
```

Comment:
this should probably still be a graceful exit like before (we already rendered the error via `prompts.cancel`)